lib/src/main/java/com/basistheory/android/view/TextElement.kt

```diff
@@ -232,4 +216,25 @@ open class TextElement : FrameLayout {
             it(event)
         }
     }
+
+    override fun onSaveInstanceState(): Parcelable {
```

Comment:
I would prefer having overrides all together and at the top, before private ones

Reply:
Easy change, I can include with the next PR